examples/carousel/carousel-1/carousel-1.html

```diff
@@ -239,7 +239,7 @@ <h2>Accessibility Features</h2>
                     When not visible, the button is positioned off-screen, instead of being hidden with <code>display:none</code>, so it is always available to screen reader users.
                     This is an important aspect of supporting users of screen readers that do not move the mouse or keyboard focus as the user is reading.
                   </li>
-                  <li>If the carousel is rotating, the button is labeled "Stop Automatic Slide Show", informing screen reader users that the slides are changing in addition to providing a way to stop the changes.</li>
+                  <li>If the carousel is rotating, the button is labeled "Stop Automatic Slide Show," informing screen reader users that the slides are changing in addition to providing a way to stop the changes.</li>
```

Comment:
@charmarkk, we may want to research this convention so we have a consistent approach. For most of eternity, the Chicago Manual of Style, which is referenced by W3C editorial guidelines, has specified that punctuation belongs inside closing quotes.

However, within recent decades, I thought some exceptions were introduced into the Chicago Manual of Style. In this case, the quotes are denoting a precise string that is used as the label. The actual label does not include the punctuation, and we would not want it to include the punctuation. This is the type of circumstance where I thought the manual provided flexibility for the position of the punctuation.

We have it both ways in the APG. It would be good to have a consistent approach going forward. It would be helpful if someone could find the relevant rules.